modules/setting/setting.go

```diff
@@ -933,8 +933,8 @@ func NewContext() {
 	newMarkup()
 
 	sec = Cfg.Section("U2F")
-	U2F.TrustedFacets, _ = shellquote.Split(sec.Key("TRUSTED_FACETS").MustString(strings.TrimRight(AppURL, "/")))
```

Comment:
Can you keep these two lines as previous otherwise it'll conflict with my PR which has these changes already.